0.data-download/scripts/data_loader.py

```diff
@@ -12,50 +12,61 @@ def load_data(data_directory, adult_or_pediatric="all"):
 
     # Define data paths
     data_directory = "../0.data-download/data/"
-    model_file = pathlib.Path(data_directory, "Model_age_column_cleaned.csv")
+    model_file = pathlib.Path(data_directory, "Model.csv")
     dependency_data_file = pathlib.Path(data_directory, "CRISPRGeneDependency.csv")
 
     # Load data
     dependency_df = (
         pd.read_csv(dependency_data_file, index_col=0).reset_index().dropna(axis=1)
     )
-    model_df = pd.read_csv(model_file, index_col=0)
-
-    # rearrange model info and gene dependency dataframe indices so DepMap_IDs are in alphabetical order
+    model_df = pd.read_csv(model_file)
+ 
+    # Ensure ID variable is ModelID; if not then rename
+    try:
+        dependency_df['ModelID']
+    except:
+        dependency_df = dependency_df.rename(columns={dependency_df.columns[0]:"ModelID"})
+
+    try:
+        model_df['ModelID']
+    except ValueError:
+        model_df = model_df.rename(columns={model_df.columns[0]:"ModelID"})
```

Comment:
I would encourage against the practice of modifying data being loaded by a function. It can get very confusing for a user! 

My comment before was suggesting to add `id_column` as a function argument:

specifically:

```python
def load_data(
    data_directory,
    adult_or_pediatric="all"
    id_column="ModelID"
    ):
    # Same function as before, but replace `ModelID` with `id_column`. Please don't hesitate to ask me questions if I'm not being clear :)
```